CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ a release.
 - SoftDeleteable: Support to use annotations as attributes on PHP >= 8.0.
 - Blameable: Support to use annotations as attributes on PHP >= 8.0.
 - IpTraceable: Support to use annotations as attributes on PHP >= 8.0.
+- SoftDeleteable: Support for Carbon type attributes
```

Comment:
I'm not a Carbon user. Are these changes only compatible with the features provided by this library or with any other custom implementation?

Reply:
Any custom implementation. But I guess Carbon is the most frequently used.

Reply:
Then, I think we should be clear about that.
Consider something like this:
>SoftDeleteable: Support for type attributes in custom `\DateTimeInterface` implementations (like [Carbon](https://carbon.nesbot.com)).

Reply:
I added a clarification. 
```Support for custom column types (like Carbon).```
These columns might be of any type, like someone could write a simple converter to store it as a string.